### ISSUE TITLE
Notification that an AJAX request has failed

### ISSUE THREAD

**Issue by copperwater** [YACS-RCOS/yacs#201]
I noticed when trying to click through to a department that the logs were showing a 500 return code, which the frontend was silently discarding because it only checks for a 200 OK and takes action in that circumstance.

There should be some sort of notification - perhaps a popup lasting for a few seconds - when an AJAX request fails to complete, which gives a brief bit of information about what went wrong.

**Comment by Bad-Science** [comment]
Yeah we definitely need this. A small banner would be nice. @YushYush Any thoughts on design?

**Comment by Bad-Science** [comment]
Let's revive this. We should have error handling for all API requests, and create some service for displaying error messages. It should be easy with the new api client.
